WORKSPACE.bzlmod

```diff
@@ -90,9 +83,15 @@ googletest_deps()
 
 http_archive(
     name = "com_google_protobuf",
-    sha256 = "e32100a8013870d24ffc37dad6781a61e5d0c99501bcb04d39c340a1c44a8e63",
-    strip_prefix = "protobuf-26.0",
-    urls = ["https://github.com/protocolbuffers/protobuf/releases/download/v26.0/protobuf-26.0.tar.gz"],
+    patch_args = ["-p1"],
+    patches = [
+        "//buildpatches:com_google_protobuf_18241.patch",
+        "//buildpatches:com_google_protobuf_18242.patch",
+        "//buildpatches:com_google_protobuf_18243.patch",
+    ],
```

Comment:
Nit: could you please add a comment here with the PR URLs? that way we can track them easier.